error_reading_dataframe = ArgumentError("\"/home/users/nticea/code/QuasiperiodicSuperconductivity/data_collection/phase_diagram/data/17L_ΦQ_LGE_2023-07-12_18:31:31.csv\" is not a valid file or doesn't exist")
Finding Tc using LGE
d-wave configuration
λ = 0.9326281137684495
I've returned
d-wave configuration
λ = 3.1243831996881273
d-wave configuration
